Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -166,19 +167,15 @@ - (void) dealloc {
 #pragma mark - Debug methods
 
 - (void)log:(NSString *)filename line:(int)line message:(NSString *)format, ... {
```

Comment:
Yes, It's probably better to remove it completely and cut over to the new interface. 
Done.